dist/src/main/java/io/camunda/application/configuration/BrokerBasedConfiguration.java

```diff
@@ -68,7 +72,7 @@ public SchedulerConfiguration schedulerConfiguration() {
     return new SchedulerConfiguration(cpuThreads, ioThreads, metricsEnabled, "Broker", nodeId);
   }
 
-  @ConditionalOnProperty(prefix = "zeebe.broker.gateway", name = "enable", havingValue = "false")
+  @ConditionalOnProperty(prefix = "camunda.rest", name = "enabled", havingValue = "false")
```

Comment:
❓ Does it mean we are not disabling it when we disable the embedded gateway? 🤔

Reply:
Yes, that's what I understood from our discussion we are okay with that, and the user must disable the REST API explicitly. But maybe I misunderstood something 🤔 

In the first iteration, I kept it with `zeebe.broker.gateway.enable`; if set to `true`, it will start the REST API. That way, starting the broker would start the `gateway-grpc` and `gateway-rest`. If set to `false`, both would not be started. When running the app with another profile, it would disable the `gateway-rest` when set to `false`.

However, I could try to combine these properties, basically, if `zeebe.broker.gateway.enable` and `camunda.rest.enabled` set to `true`, then enable the REST API when the broker is started.

Reply:
I adjusted the condition with 58cb2d9996b91f2a80f6e08800a0e92d93c0bdb2

When running the broker:
* `zeebe.broker.gateway.enable=false` then the embedded gateway is disabled, including the REST API
* `zeebe.broker.gateway.enable=true` and `camunda.rest.enabled=true` (default=`true`), then the embedded gateway is enabled, including the REST API
* `zeebe.broker.gateway.enable=true` and `camunda.rest.enabled=false, then the embedded gateway is enabled but the the REST API is disabled

When running without the broker
* `camunda.rest.enabled=true`, the REST API is started
* `camunda.rest.enabled=false`, the REST API is disabled
* When `zeebe.broker.gateway.enable` is set to `false`, the REST API is also disabled. However, if not present, the default value `true` is set, which enables the REST API.

An alternative would be implementing an initializer to handle the different cases when bootstrapping the app. But I felt this might be sufficient, and the last case seems to be an edge case.